flygym/mujoco/core.py

```diff
@@ -265,6 +265,7 @@ def __init__(
         contact_sensor_placements: List = preprogrammed.all_tarsi_links,
         output_dir: Optional[Path] = None,
         arena: BaseArena = None,
+        xml: str = "mjcf_model",
```

Comment:
Would the main `NeuroMechFly` class ever require loading the reference XML for seqIK? If not, we can hard code it here. This can be confusing.

Otherwise, we should add info on the `xml` argument in the documentation.